docker/types.go

```diff
@@ -108,6 +120,22 @@ type Config struct {
 	Shell           strslice.StrSlice   `json:",omitempty"` // Shell for shell-form of RUN, CMD, ENTRYPOINT
 }
 
+// github.com/docker/distribution/manifest/schema1/config_builder.go
```

Comment:
I think this location may have changed to github.com/docker/distribution/blob/master/manifest/schema1/config_builder.go.  At least I'm finding the file there (added blob/master) and not at this location on the web.